src/client/cockpit-client

```diff
@@ -75,6 +82,9 @@ class CockpitClient(Gtk.Application):
 
         self.uri = self.ws.start()
 
+    def new_window(self, *args):
+        CockpitClientWindow(self, self.uri).present()
+
     def do_activate(self):
         CockpitClientWindow(self, self.uri).present()
```

Comment:
These two do exactly the same, could new-window just be bound to do_activate()?

Reply:
no.  the signatures are different.

Reply:
`*args` is compatible to "arguments"